### PR TITLE
[WIP] Fix recording output format

### PR DESCRIPTION
**Pull request by mithro**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -9,8 +9,8 @@ test:
 	make style unittests integration
 
 #	TODO: Use configure to detect Python versions currently installed
-	make PYTHONVERSION=2.7 -Cpython-api style unittests integration
-	make PYTHONVERSION=3.4 -Cpython-api style unittests integration
+	cd python-api; make PYTHONVERSION=2.7 style unittests integration
+	cd python-api; make PYTHONVERSION=3.4 style unittests integration
```
Comment (MaZderMind) [inline Makefile.am]: why exactly are you changing this?

File: run-demo.sh [hunk 2]
```diff
@@ -7,6 +7,7 @@ AUDIO_CAPS="audio/x-raw, rate=48000, channels=2, format=S16LE, layout=interleave
 
 killall gst-switch-srv || true
 sleep 2
+#GST_DEBUG=*:4 ./tools/gst-switch-srv -f "$VIDEO_CAPS" -r &
```
Comment (MaZderMind) [inline run-demo.sh]: you should be able to run sth. like
`GST_DEBUG=*:4 sh run-demo.sh`
to set that on the fly

File: Makefile.am (hunk 1 above)
Comment (mithro) [inline Makefile.am]: Oops that should be in a different pull request. See the comment on the commit https://github.com/timvideos/gst-switch/commit/bd709218fd59307643c80e728171b0a8b895d166

File: run-demo.sh (hunk 2 above)
Comment (mithro) [inline run-demo.sh]: Problem is that makes the gst-launch commands noisy to. Not planning on committing this bit.
